public/pages/Indices/components/ShrinkIndexFlyout/ShrinkIndexFlyout.tsx

```diff
@@ -145,7 +145,7 @@ export default class ShrinkIndexFlyout extends Component<ShrinkIndexProps, Shrin
       }
 
       const indexWriteBlock = get(sourceIndexSettings, [sourceIndex.index, "settings", INDEX_BLOCKS_WRITE_SETTING]);
-      if (!indexWriteBlock) {
+      if (indexWriteBlock != "true" && indexWriteBlock != true) {
```

Comment:
use `!==` to indicate not equal will be safer

Reply:
Thanks, I've made some change yet, can you help to take a look?